app/window.go

```diff
@@ -185,6 +185,9 @@ func (w *Window) processFrame(frameStart time.Time, size image.Point, frame *op.
 	case router.TextInputClose:
 		go w.Run(func() { w.driver.ShowTextInput(false) })
 	}
+	if hint, ok := w.queue.q.TextInputHint(); ok {
+		w.driver.SetInputHint(hint)
```

Comment:
Use a goroutine and Window.Run like for ShowTextInput above. See also https://gioui.org/commit/8611894b4bb36629b1255ee30989d38438274fe9.